### DISCUSSION TITLE
Web flasher progress indicator broken?

### DISCUSSION THREAD

**Discussion by erikpuntnl** [darkxst/silabs-firmware-builder#157] [General]
When using the web flasher the progress indicator stays at 0.0%.
It does it on the Chrome and Edge browser on Windows.
Tried three different laptops, all the same issue.

**Comment by darkxst** [comment]
There are driver issues on windows. See this comment for workarounds.
https://github.com/darkxst/silabs-firmware-builder/issues/107#issuecomment-2454076415

**Reply by erikpuntnl** [reply]
Thank you!
